test/fuzz/basic.ml

```diff
@@ -29,10 +29,16 @@ let limits =
 
 let table_type = pair limits ref_type
 
-let final = choose [ const No_final; const Final ]
+let final = const Final
+(* TODO: complete No_final *)
 
-let elem_mode = const Elem_declarative (* for functions *)
-(* TODO: complete Elem_active / Elem_passive *)
+let elem_active =
+  let* ind = const (None : indice option) in
+  let+ expr = const [] in
+  Elem_active (ind,expr)
+
+let elem_mode = const Elem_passive
```

Comment:
I'm not sure we want to do it this way.

You should have an `elem` generator that would simply be:

```ocaml
let elem =
  choose [ elem_active; elem_passive; elem_declarative ]
```

Reply:
Yes, of course. But for the moment, i consider only passive elements.

Reply:
in the commit below [c22f23b](https://github.com/OCamlPro/owi/pull/28/commits/c22f23b88e268a2bcf33f6db09607814a2ed51ac), I've added `elem_active`